src/lib/route-builder.js

```diff
@@ -176,8 +176,8 @@ class RouteBuilder {
       ilpPacket.account, ilpPacket.amount)
 
     const sourceLedger = sourceTransfer.ledger
-    const nextHop = yield this.quoter.findBestPathForSourceAmount(
```

Comment:
Is `findBestPathForSourceAmount` still used or was this the only call? If its the latter, the function should be removed.